x-pack/test_serverless/functional/test_suites/search/custom_role_access.ts

```diff
@@ -53,6 +53,8 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       if (roleAuthc) {
         await samlAuth.invalidateM2mApiKeyWithRoleScope(roleAuthc);
       }
+      // delete custom role
+      await samlAuth.deleteCustomRole();
```

Comment:
we need to delete role, because platform security tests validate security roles UI and roles list.